plugins/aws_sts/test/samson_aws_sts/samson_plugin_test.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+require_relative '../test_helper'
+
+SingleCov.covered! uncovered: 5
```

Comment:
how about 0 ? :trollface:

Reply:
all the important stuff is tested ;)

Reply:
Still prefer 0, the other plugins can serve as a copy-paste example, should be pretty straight forward, if you run into any trouble let me know and I'll help out